chromium_src/chrome/browser/autocomplete/chrome_autocomplete_provider_client.cc

```diff
@@ -60,10 +62,16 @@ void ChromeAutocompleteProviderClient::OpenLeo(const std::u16string& query) {
       browser->tab_strip_model()->GetActiveWebContents());
   DCHECK(chat_tab_helper);
 
+  auto* conversation_handler =
+      ai_chat::AIChatServiceFactory::GetForBrowserContext(profile_)
```

Comment:
don't we need a null check here before calling `GetOrCreateConversationHandlerForContent`?

Reply:
It was covered by
```
  CHECK(profile_->IsRegularProfile());
```
and the feature not being added to the omnibox suggestions in incognito profiles, but I agree it's better to check the service - added in 5f890855da16e83c3b3cf60a4b60937b88ed17a3